editor/editor_node.cpp

```diff
@@ -4932,7 +4932,10 @@ void EditorNode::_save_open_scenes_to_config(Ref<ConfigFile> p_layout) {
 	p_layout->set_value(EDITOR_NODE_CONFIG_SECTION, "open_scenes", scenes);
 
 	String currently_edited_scene_path = editor_data.get_scene_path(editor_data.get_edited_scene());
-	p_layout->set_value(EDITOR_NODE_CONFIG_SECTION, "current_scene", currently_edited_scene_path);
+	// don't save a bad path to the config
```

Comment:
This is the preferred comment style in the codebase (capital letter to start and ends with a dot).
```suggestion
	// Don't save a bad path to the config.
```